quickstart/php/sms/example-1/sendnotifications.5.x.php

```diff
@@ -0,0 +1,55 @@
+<?php
+  /* Send an SMS using Twilio. You can run this file 3 different ways:
+   *
+   * 1. Save it as sendnotifications.php and at the command line, run 
+   *        php sendnotifications.php
+   *
+   * 2. Upload it to a web host and load mywebhost.com/sendnotifications.php 
+   *    in a web browser.
+   *
+   * 3. Download a local server like WAMP, MAMP or XAMPP. Point the web root 
+   *    directory to the folder containing this file, and load 
+   *    localhost:8888/sendnotifications.php in a web browser.
+   */
+
+  // Step 1: Get the Twilio-PHP library from twilio.com/docs/libraries/php, 
+  // following the instructions to install it with Composer.
+  require_once "vendor/autoload.php";
+
+  // Step 2: set our AccountSid and AuthToken from https://twilio.com/console
+  $AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+  $AuthToken = "your_auth_token";
+
+  // Step 3: instantiate a new Twilio Rest Client
+  $client = new Twilio\Rest\Client($AccountSid, $AuthToken);
```

Comment:
You can use `use Twilio\Rest\Client` on line 18 to avoid creating the new instance using the whole namespace.

```
require_once "vendor/autoload.php";
use Twilio\Rest\Client;

// Step 2: set our AccountSid and AuthToken from https://twilio.com/console
$AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$AuthToken = "your_auth_token";

// Step 3: instantiate a new Twilio Rest Client
 $client = new Client($AccountSid, $AuthToken);
```